consensus/core/src/commit_syncer.rs

```diff
@@ -308,6 +308,19 @@ impl<C: NetworkClient> CommitSyncer<C> {
                             missing, fetched_commit_range
                         );
                     }
+                    for block_ref in missing {
```

Comment:
We do expect `missing` to be empty with `add_certified_commit()`, right?

Reply:
Exactly! That metric is temporary for this testnet release. Once the commit syncer improvements are in it won't be relevant anymore (and I'll remove it then)